### PR TITLE
Added events past Oct 22, updated officers, updated HoTs, updated members

### PR DESCRIPTION
**Pull request by wjiang42**
Added:
 - AB Spring Concert - COIN and Bren Joy
 - AB Speakers - Stephanie Hsu
 - AB Speakers - Jennette McCurdy
 - AB Comedy - Melissa Villasenor
 - UE - Allegheny County Executive Debate
 - AB Winter Concert - Tiffany Day

### CODE REVIEW

File: _data/people_currentcrew_officers.json [hunk 1]
```diff
@@ -1,72 +1,79 @@
...
+        "andrewid": "maristey",
+        "img": "maristey.jpg",
         "title": "Co-Head of Tech",
-        "desc": "Shannon is a senior studying computer science with a minor in music technology. She does sound and production management in AB Tech. In her spare time, she enjoys audiobooks, trying to play instruments, and long walks (at sunset)."
+        "desc": "Stella is a junior studying mechanical engineering. In AB Tech, she does a lot of sound and driving trucks. Sometimes she will randomly do stage management, get a new tattoo, or blast K-pop in techroom while battling balloons. "
     },
     {
-        "name":"Alex Strasser",
-        "andrewid": "astrasse",
-        "img": "astrasse.jpg",
```
Comment (DaAwesomeP) [inline _data/people_currentcrew_officers.json:19]: Please also remove the old unused officer photos

File: _data/people_currentcrew_officers.json [hunk 2]
```diff
@@ -1,72 +1,79 @@
...
-        "title": "Co-Head of Tech",
-        "desc": "Alex Strasser is a senior studying ECE and Robotics. He enjoys repairing our equipment, doing lighting for events, and working with event organizers to make events meet their dreams!"
+        "name":"Queenie Chen",
+        "andrewid": "siyanche",
+        "img": "siyanche.jpg",
+        "title": "Co-Finance Chair",
+        "desc": "I'm Queenie! Junior in Computational Finance minoring in Drama and Behavioral Economics. I love swimming, hiking, participating in theatre (watching/ producing!). Fun fact about me -- my birthday is usually the first/ second day of class for spring semester."
```
Comment (DaAwesomeP) [inline _data/people_currentcrew_officers.json:21]: Extra spaces after the slashes

File: _data/people_currentcrew_officers.json [hunk 3]
```diff
@@ -1,72 +1,79 @@
...
+    },
+    {
+        "name": "Maggie Gong",
+        "andrewid": "yiyugong",
+        "img": "yiyug.jpg",
```
Comment (DaAwesomeP) [inline _data/people_currentcrew_officers.json:26]: New images need to be added/uploaded to the pull

**Review by DaAwesomeP — CHANGES_REQUESTED** [review]
Just some minor things! Mainly missing the officer photo assets.

File: _data/people_currentcrew_officers.json (hunk 2 above)
Comment (wjiang42) [reply _data/people_currentcrew_officers.json:21]: Do you mean dashes (--)?

**Comment by DaAwesomeP** [comment]
@wjiang42 Let me know when you're ready for another review by clicking the little circle-arrow icon!

**Comment by wjiang42** [comment]
Did I request a review? I clicked some buttons and do not know if they worked

**Review by DaAwesomeP — APPROVED** [review]

**Pull request merged**